### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by lsoucasse**
Introduce two simple tests for Mors that check results for random input for both Spada and Baraffe models.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          echo $'\n```' >> $GITHUB_STEP_SUMMARY
+          coverage report >> $GITHUB_STEP_SUMMARY
+          echo $'\n```' >> $GITHUB_STEP_SUMMARY
+
+      - name: Make coverage badge
+        if: ${{ github.ref == 'refs/heads/master' && matrix.python-version == '3.10' }}
+        uses: schneegans/dynamic-badges-action@v1.7.0
+        with:
+          auth: ${{ secrets.GIST_TOKEN }}
+          gistID: 99391a66bb9229771504c3a4db611d05
```
Comment (stefsmeets) [inline .github/workflows/tests.yaml:70]: The way this works is that you make a gist, does not matter what is in there, and copy the gist ID. The action will update the badge if succesful, this ID points to JANUS: https://gist.github.com/stefsmeets/99391a66bb9229771504c3a4db611d05

Then generate a new token scoped to modifying gists, named GIST_TOKEN: https://github.com/settings/tokens

Then you update the GIST_TOKEN in https://github.com/FormingWorlds/MORS/settings/secrets/actions under Repository secrets. Ask @timlichtenberg to give you permissions (I don't have them).



```suggestion
          gistID: your own
```

File: src/mors/stellarevo.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def _shouldCompileNew(starEvoDir,evoModels):
     compileNew = True
 
     # Check if previously compiled models already exist
-    if ( os.path.isfile(starEvoDir+evoModels+".pickle") ):
+    if ( os.path.isfile(starEvoDir+"/"+evoModels+".pickle") ):
```
Comment (stefsmeets) [inline src/mors/stellarevo.py:197]: Nice fix, I ran into the same issue today 🚀

**Review by stefsmeets — CHANGES_REQUESTED** [review]
Nice, looks good! Also works well on my PC.

Just need to fix the coverage badge. And then don't forget to add the badge to the readme. 😅

E.g. https://github.com/FormingWorlds/JANUS/blob/d289631fe2b6322a27fda31d41576998f2093680/README.md?plain=1#L2

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (lsoucasse) [reply .github/workflows/tests.yaml]: Gist is created and ID updated in README and tests.yaml.
I still need to create associated token and store it in the repo but this is outside this PR.

**Review by stefsmeets — APPROVED** [review]
Looks good to me! 🚀

**Pull request merged**